src/php/packages/renderer/README.md

```diff
@@ -0,0 +1,72 @@
+# Getting started
+
+**Recommended versions of required software:**
+
+- PHP >= 7.x
+- Composer >= v1.8.4
+
+## Introduction
+
+Files overview:
+
+- `composer.json`: contains information about PHP dependencies, similar to `package.json`
+- `composer.lock`: lock file with dependencies, similar to `yarn.lock`
+- `bootstrap.php`: instantiates `$twig`
+- `render.php`: uses `$twig` and renders templates
+
+## Settings
+
+Please select the ECL system you want to work with:
+
+```bash
+export ECL_SYSTEM=ec
+```
+
+## Installation
+
+Either way:
+
+```shell
+$ yarn setup
+```
+
+```shell
+$ composer run setup
+```
+
+Note that the `npm` scripts facilitate integration between tools in Node.js and PHP. Scripts naming may differ because of reserved keywords, such as `install` in composer, which is replaced by `setup`.
+
```

Comment:
We should also have a paragraph here to explain how to clean the data from previous build (as in getting-started.md)